code/unit_tests/map_tests.dm

```diff
@@ -496,26 +496,6 @@
 
 //=======================================================================================
 
-/datum/unit_test/simple_pipes_shall_not_face_north_or_west // The init code is worthless and cannot handle it
```

Comment:
Was this being readded somewhere? Not sure if that got done yet.

Reply:
I will readd this following Psi's comment above. It is no longer broken to have pipes facing north or west, but we can continue enforcing this for consistency.